commit 6175507c06de56e03407004bd2f289ed2cce034d
Author: Florian Weimer <fweimer@redhat.com>
Date:   Fri Feb 1 14:04:42 2019 +0100

    support: Correct error message for TEST_COMPARE_STRING
    
    It should say "string", not "blob".

diff --git a/support/support_test_compare_string.c b/support/support_test_compare_string.c
index d6d6d551de3d8ad6..36b248c9c196e0f3 100644
--- a/support/support_test_compare_string.c
+++ b/support/support_test_compare_string.c
@@ -75,7 +75,7 @@ support_test_compare_string (const char *left, const char *right,
       || memcmp (left, right, left_length) != 0)
     {
       support_record_failure ();
-      printf ("%s:%d: error: blob comparison failed\n", file, line);
+      printf ("%s:%d: error: string comparison failed\n", file, line);
       if (left_length == right_length && right != NULL && left != NULL)
         printf ("  string length: %zu bytes\n", left_length);
       else
diff --git a/support/tst-test_compare_string.c b/support/tst-test_compare_string.c
index ec2ae128f9f937b9..f471a8b6a171d548 100644
--- a/support/tst-test_compare_string.c
+++ b/support/tst-test_compare_string.c
@@ -60,22 +60,22 @@ do_test (void)
 
   TEST_VERIFY
     (strcmp (proc.out.buffer,
-"tst-test_compare_string.c:29: error: blob comparison failed\n"
+"tst-test_compare_string.c:29: error: string comparison failed\n"
 "  left string: 0 bytes\n"
 "  right string: NULL\n"
-"tst-test_compare_string.c:30: error: blob comparison failed\n"
+"tst-test_compare_string.c:30: error: string comparison failed\n"
 "  left string: 1 bytes\n"
 "  right string: 0 bytes\n"
 "  left (evaluated from \"X\"):\n"
 "      \"X\"\n"
 "      58\n"
-"tst-test_compare_string.c:31: error: blob comparison failed\n"
+"tst-test_compare_string.c:31: error: string comparison failed\n"
 "  left string: NULL\n"
 "  right string: 1 bytes\n"
 "  right (evaluated from \"X\"):\n"
 "      \"X\"\n"
 "      58\n"
-"tst-test_compare_string.c:32: error: blob comparison failed\n"
+"tst-test_compare_string.c:32: error: string comparison failed\n"
 "  string length: 4 bytes\n"
 "  left (evaluated from \"abcd\"):\n"
 "      \"abcd\"\n"
@@ -83,13 +83,13 @@ do_test (void)
 "  right (evaluated from \"abcD\"):\n"
 "      \"abcD\"\n"
 "      61 62 63 44\n"
-"tst-test_compare_string.c:33: error: blob comparison failed\n"
+"tst-test_compare_string.c:33: error: string comparison failed\n"
 "  left string: 4 bytes\n"
 "  right string: NULL\n"
 "  left (evaluated from \"abcd\"):\n"
 "      \"abcd\"\n"
 "      61 62 63 64\n"
-"tst-test_compare_string.c:34: error: blob comparison failed\n"
+"tst-test_compare_string.c:34: error: string comparison failed\n"
 "  left string: NULL\n"
 "  right string: 4 bytes\n"
 "  right (evaluated from \"abcd\"):\n"
